aio_pika/patterns/rpc.py

```diff
@@ -115,6 +116,11 @@ def create_future(self) -> Tuple[asyncio.Future, str]:
         future.add_done_callback(self.__remove_future(correlation_id))
         return future, correlation_id
 
+    def _routing_key(self, method_name: str) -> str:
+        return f'{self.rpc_exchange.name}::{method_name}' \
```

Comment:
I don't like that such an odd way of formatting has been chosen here. I would start with the usual way of formatting here, and you, in your library, have already inherited and overridden one method as you need it. BTW `_format_routing_key(self, method_name: str) -> str` should be better name for this IMHO.

Reply:
Aside from changing the function descriptor to `_format_routing_key(self, method_name: str) -> str`, how do you suggest to change the formatting of of exchange name and `method_name`? I cannot use `.` in between because `method_name` might also contain `.`. I am open to any suggestions...

Reply:
In any case, I updated the function descriptor